Tree/readme.md

```diff
@@ -47,9 +47,11 @@ that, this property should be satisfied at every node in the tree.
 * Sorting the elements of binary search tree and many more
 
 ## Questions :
-* Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp)
+* Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp) 
+* Check Whether given Binary Tree is BST ----> [C++](/Code/C++/check_tree_is_BST.cpp)
```

Comment:
`Check if a Binary Tree is a valid Binary Search Tree ----> [Java](/Code/Java/check_valid_BST.java)`

This is a previous merged pr take a pull and update readme

Reply:
this is to be done to resolve the conflict